### ISSUE TITLE
Document SerializerBuilder 

### ISSUE THREAD

**Issue by aersamkull** [aaubry/YamlDotNet#370]
Hi there

Thanks for this library. I think to gain even more popularity it would be good to include some more samples in the read me, especially for the Serialization (SerializerBuilder )

**Comment by aaubry** [comment]
I agree, the documentation really needs some love. Alas, I don't have much time to triage the issues, much less to work on the documentation. But help is always welcome ! ;)
